### ISSUE TITLE
Need a big red button for package building

### ISSUE THREAD

**Issue by igormunkin** [tarantool/tarantool#5416]
There is a [doc](https://github.com/tarantool/tarantool/wiki/Build-RPM-or-Deb-package-using-packpack) about building binary packages. The recipe contains a bunch of peculiarities, so such tedious and jogtrot activity becomes a problem (e.g. the building a Debug package). It would be nice to have a big red button to build the package from the particular patch. As a result it would be nice to have the script with the following interface:
```
$ ./build.pl --build Debug --dist=el7 --commit a696353 ~/tarantool-src
https://transfer.sh/imun9/tarantool-1.10.7.13-1.el7.x86_64.rpm
```

**Comment by Totktonada** [comment]
Possible variant: a manual job in CI, which builds a package and holds it in artifacts.

**Comment by igormunkin** [comment]
> Possible variant: a manual job in CI, which builds a package and holds it in artifacts.

@Totktonada, yeah, that's also a good one, except I prefer it being triggered via the branch naming (similar to the way `-full-ci` suffix works now).

**Comment by Totktonada** [comment]
NB: Think about a repository that corresponds a particular branch.

**Comment by igormunkin** [comment]
> NB: Think about a repository that corresponds a particular branch.

Don't get this one. If you're talking about git repositories (or GitHub forks) this looks inconvenient: too many buttons to click, so I'll preferably stay on struggling with using docker images and spec patching. If you propose to create a standalone distribution repo for particular branch, this looks excess a bit, since I doubt anyone will enable/disable repo to install the one patched version. At the same time, there are some cases when patched versions are used really often -- for such cases it looks like the most convenient approach.

Anyway, as I've already mentioned above, a simple Perl/bash/etc script fits me more than fine: I'm just tired of doing lot's of similar actions manually again and again for building the custom package. Implementing such script in a fortnight would make me more happy than a super-duper spaceship with unpretty web interface in a half year.
